src/pages/components/cards.mdx

```diff
@@ -146,6 +146,13 @@ take the user to different pages. Static cards can appear with or without a bord
 
 </Row>
 
+#### Logo
+
```

Comment:
One recommendation would be to put this at the top of the Modifiers section, to keep the modifiers in alphabetical order.